turbopack/crates/turbo-persistence/src/static_sorted_file.rs

```diff
@@ -109,6 +109,11 @@ impl StaticSortedFile {
         let mmap = unsafe { Mmap::map(&File::open(&path)?)? };
         #[cfg(unix)]
         mmap.advise(memmap2::Advice::Random)?;
+        #[cfg(unix)]
+        {
+            let offset = meta.block_offsets_start(mmap.len());
+            let _ = mmap.advise_range(memmap2::Advice::WillNeed, offset, mmap.len() - offset);
```

Comment:
I'm not really sure, but maybe this should be `::Random`?
But that seems to imply that the OS should do less readahead? Unclear.

https://docs.rs/memmap2/latest/memmap2/enum.Advice.html#variant.Random